<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DefMatriz</class>
 <widget class="QDialog" name="DefMatriz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qual o tamanho?</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_x">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_y">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_z">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DefMatriz</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DefMatriz</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
